packages/wrangler/src/__tests__/dev.test.ts

```diff
@@ -156,6 +149,11 @@ describe.sequential("wrangler dev", () => {
 		} catch (e) {
 			console.error(e);
 		}
+		if (spy.mock.calls.length === 0) {
+			throw new Error(
+				"Config was never reached:\n" + JSON.stringify(std, null, 2)
+			);
+		}
```

Comment:
Previously if reading the config was never reached in the Wrangler run, we would just get a useless error on the return statement below.
Now we dump out the stdout etc so we can diagnose the problem.